packages/office-ui-fabric-react/src/components/Facepile/Facepile.styles.ts

```diff
@@ -1,17 +1,154 @@
 import { IFacepileStyleProps, IFacepileStyles } from './Facepile.types';
 
-export const getStyles = (props: IFacepileStyleProps): IFacepileStyles => {
-  const { className } = props;
+import {
+  IStyle,
+  hiddenContentStyle,
+  getFocusStyle,
+  getGlobalClassNames,
+} from '../../Styling';
 
-  return {
+const GlobalClassNames = {
+  root: 'ms-Facepile',
+  addButton: 'ms-Facepile-addButton ms-Facepile-itemButton',
+  descriptiveOverflowButton: 'ms-Facepile-descriptiveOverflowButton ms-Facepile-itemButton',
+  itemButton: 'ms-Facepile-itemButton ms-Facepile-person',
+  itemContainer: 'ms-Facepile-itemContainer',
+  members: 'ms-Facepile-members',
+  overflowButton: 'ms-Facepile-overflowButton ms-Facepile-itemButton',
+};
+
+export const styles = (
+  props: IFacepileStyleProps
+): IFacepileStyles => {
+  const {
+    className,
+    theme,
+    spacingAroundItemButton = 2,
+  } = props;
+
+  const { palette, fonts } = theme;
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  const ItemButtonStyles: IStyle = {
+    textAlign: 'center',
+    padding: 0,
+    borderRadius: '50%',
+    verticalAlign: 'top',
+    selectors: {
+      'button&': {
```

Comment:
is there a need for this specificity? (I'm not completely sure.)

Reply:
There are divs with the `ms-Facepile-itemButton` class as well as buttons with the `ms-Facepile-itemButton` class. Only the buttons with that class get these `'button&'` styles applied. This is just replicating the styles that were previously in the scss file and it appears to be working as intended.